docs/4.0/docs/examples/sealctl-registry.md

```diff
@@ -0,0 +1,226 @@
+  -h, --help            help for images
+      --json            output in JSON format
+  -n, --name string     registry name (default "sealos.hub:5000")
+
+Global Flags:
+      --debug                   enable debug logger
+      --root string             storage root dir (default "/var/lib/containers/storage")
+      --runroot string          storage state dir (default "/run/containers/storage")
+      --show-path               enable show code path
+      --storage-driver string   storage-driver (default "overlay")
+      --storage-opt strings     storage driver option
+```
+
+#### How to using
+
+```shell
+root@default-golang-0:~# sealctl  registry  images -n 192.168.64.63:5000 --filter name=kube*,tag=*
++--------------------+-------------------------+---------+-------------------------------------------------------------------------+
+| RegistryName       | ImageName               | Tag     | ImageID                                                                 |
++--------------------+-------------------------+---------+-------------------------------------------------------------------------+
+| 192.168.64.63:5000 | kube-apiserver          | v1.19.0 | sha256:522d17d35a8994637d27d1232bebd35cfae8e3e21ab359431403f2b8023e332c |
+| 192.168.64.63:5000 | kube-controller-manager | v1.19.0 | sha256:6c11a3d4d06385f7547a5ea0c3f0d5e7b12baa558111d01406ac1f778cb3f00b |
+| 192.168.64.63:5000 | kube-proxy              | v1.19.0 | sha256:c752ecbd04bc4517168a19323bb60fb45324eee1e480b2b97d3fd6ea0a54f42d |
+| 192.168.64.63:5000 | kube-scheduler          | v1.19.0 | sha256:529a1566960a5b3024f2c94128e1cbd882ca1804f222ec5de99b25567858ecb9 |
++--------------------+-------------------------+---------+-------------------------------------------------------------------------+
+2023-01-01T12:31:49 info Image count 4
+2023-01-01T12:31:49 info Images Version count 4
+```
+
+filter logic:
+
+filter key support: name, tag 
+
+expressions:
+
+  `*val`: suffix 
+  `val*`: prefix
+  `value`: equals
+  `<none>`: empty
+  `*val*`: contains
+
+
+
+## Remove registry image 
+
+```
+registry rmi image
+
+Usage:
+  sealctl registry rmi [flags]
+
+Examples:
+sealctl registry  rmi labring/lvscare:v4.1.3
+
+Flags:
+  -h, --help          help for rmi
+  -n, --name string   registry name (default "sealos.hub:5000")
+
+Global Flags:
+      --debug                   enable debug logger
+      --root string             storage root dir (default "/var/lib/containers/storage")
+      --runroot string          storage state dir (default "/run/containers/storage")
+      --show-path               enable show code path
+      --storage-driver string   storage-driver (default "overlay")
+      --storage-opt strings     storage driver option
+```
+
+### How to using
+
+```shell
+sealctl  registry  rmi 192.168.64.63:5000/kube-apiserver:v1.19.0 -n 192.168.64.63:5000
```

Comment:
We should either use:

<pre>
```shell
$ sealctl <commands> ...
```
</pre>

Or:

<pre>
```shell
sealctl <commands> ...
```
</pre>

And should not use them both.